### PR TITLE
[EDA] Relationship Settings: Allow Auto-Created Duplicate Relations

### PR DESCRIPTION
**Pull request by skosgi1**
# Critical Changes

# Changes

# Issues Closed

# New Metadata
**Custom Labels**
1) stgDuplicateRelationships
2) stgDuplicateRelationshipsHelp

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: force-app/main/default/classes/RelationshipSettingsVMapper.cls [hunk 1]
```diff
@@ -76,7 +76,10 @@ public virtual with sharing class RelationshipSettingsVMapper {
             relationshipTypeComboboxValueVModelsList
         );
 
-        return new RelationshipSettingsVModel(defaultRelationshipComboboxVModel);
+        return new RelationshipSettingsVModel(
+            defaultRelationshipComboboxVModel,
+            !currentRelationshipSettingsModel.allowAutoCreatedDuplicates
```
Comment (skosgi1) [inline force-app/main/default/classes/RelationshipSettingsVMapper.cls:81]: @MatthewBlanski would this be the right place to handle the inversion of the custom settings value?

File: force-app/main/default/lwc/relationshipSettings/relationshipSettings.js [hunk 2]
```diff
@@ -60,34 +70,45 @@ export default class relationshipSettings extends LightningElement {
...
+
+        this.template.querySelector("c-settings-save-canvas").handleHierarchySettingsChange(hierarchySettingsChange);
+    }
+
+    handleAuoCreatedDuplicatesChange(event) {
+        const eventDetail = event.detail;
+        let hierarchySettingsChange = {
+            settingsType: "boolean",
+            settingsName: "Allow_AutoCreated_Duplicates__c",
+            settingsValue: !eventDetail.value, // Save the inverse as UI display the inverse value of the settings
```
Comment (xinnanlisf) [inline force-app/main/default/lwc/relationshipSettings/relationshipSettings.js:89]: lets move the comment to the top

File: force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 3]
```diff
@@ -1462,6 +1462,22 @@
         <shortDescription>Label for the Require Preferred Phone setting</shortDescription>
         <value>Require Preferred Phone</value>
     </labels>
+    <labels>
+        <fullName>stgDuplicateRelationships</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Label for the Relationship Duplicates Settings</shortDescription>
+        <value>Duplicate Relationships</value>
```
Comment (xinnanlisf) [inline force-app/main/default/labels/CustomLabels.labels-meta.xml:1471]: Suggest we would name this as Duplicate Relationships Prevention. Pending doc's response.

**Review by xinnanlisf — CHANGES_REQUESTED** [review]

File: force-app/main/default/lwc/relationshipSettings/relationshipSettings.html [hunk 4]
```diff
@@ -10,6 +10,7 @@
             >
                 <!-- Body of Relationship Settings page (populates Slot in child component) -->
                 <template if:true={relationshipSettingsVModel}>
+                    <!-- Reciprocal Method Mark Up-->
                     <div class="slds-grid slds-grid_vertical-align-start slds-wrap">
```
[the remark this answers is not in the document]
Comment (skosgi1) [reply force-app/main/default/lwc/relationshipSettings/relationshipSettings.html]: you're right, remove it.

**Review by xinnanlisf — APPROVED** [review]

**Pull request merged**